docs/en/ingest-management/elastic-agent/example-kubernetes-fleet-managed-agent-helm.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[example-kubernetes-fleet-managed-agent-helm]]
+= Example: Install {fleet}-managed {agent} on Kubernetes using Helm
+
+preview::[]
+
+This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather Kubernetes metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
+
+For an overview of the {agent} Helm chart and its benefits, refer to <<install-on-kubernetes-using-helm>>.
+
+This guide will take you through the following steps:
+
+* <<agent-fleet-managed-helm-example-install-agent>>
+* <<agent-fleet-managed-helm-example-install-integration>>
+* <<agent-fleet-managed-helm-example-tidy-up>>
+
+
+[discrete]
+[[agent-fleet-managed-helm-example-prereqs]]
+=== Prerequisites
+
+To get started, you need:
+
+* A local install of the link:https://helm.sh/[Helm] Kubernetes package manager.
+* An link:{ess-trial}[{ecloud}] hosted {es} cluster on version 8.16 or higher.
+* An active Kubernetes cluster.
+* A local clone of the link:https://github.com/elastic/elastic-agent/tree/8.16[elastic/elastic-agent] GitHub repository. Make sure to use the `8.16` branch to ensure that {agent} has full compatibility with the Helm chart.
+
+[discrete]
+[[agent-fleet-managed-helm-example-install-agent]]
+=== Install {agent}
+
+. Open your {ecloud} deployment, and from the navigation menu select **Fleet**.
+. From the **Agents** tab, select **Add agent**.
+. In the **Add agent** UI, specify a policy name and select **Create policy**. Leave the **Collect system logs and metrics** option selected.
+. Scroll down in the **Add agent** flyout to the **Install Elastic Agent on your host** section.
+. Select the **Linux TAR** tab and copy the values for `url` and `enrollment-token`. You'll use these when you run the `helm install` command.
+. Open a terminal shell and change into a directory in your local clone of the `elastic-agent` repo.
+. Copy the following command. Before running it you'll need to make a couple of changes.
```

Comment:
```suggestion
. Copy this command:  
```

Consider moving this sentence to after the command:

"Before running it you'll need to make a ~couple of~ **few** changes."  
(Unless it's literally two changes to be made.)